feat: add tenor domains to LinkPreviewWhitelist

stefandunca opened the pull request:
Updates: #6829

stefandunca committed:
feat: add tenor domains to LinkPreviewWhitelist

Also forward the tenor URLs as previews

Updates: #6829

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://status.im/docs/))?
- [ ] Have you tested changes with mobile?
- [ ] Have you tested changes with desktop?

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (3)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 8b825283 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2832/1/) | 2022-08-31 17:29:00 | ~10 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-220831-171850-8b8252-pr2832.zip) |
| :heavy_check_mark: | 8b825283 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2832/1/) | 2022-08-31 17:40:23 | ~21 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-220831-171850-8b8252-pr2832.aar) |
| :heavy_check_mark: | 8b825283 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2832/1/) | 2022-08-31 17:41:12 | ~22 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-220831-171850-8b8252-pr2832.zip) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 61a51799 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2832/2/) | 2022-09-01 11:46:26 | ~4 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-220901-114145-61a517-pr2832.zip) |
| :heavy_check_mark: | 61a51799 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2832/2/) | 2022-09-01 11:49:33 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-220901-114145-61a517-pr2832.zip) |
| :heavy_check_mark: | 61a51799 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2832/2/) | 2022-09-01 11:51:22 | ~9 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-220901-114141-61a517-pr2832.aar) |
| | | | | | | |
| :heavy_check_mark: | 70111b2a | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-2832/3/) | 2022-09-01 17:52:19 | ~2 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-desktop-220901-174942-70111b-pr2832.zip) |
| :heavy_check_mark: | 70111b2a | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-2832/3/) | 2022-09-01 17:53:01 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-220901-174941-70111b-pr2832.zip) |
| :heavy_check_mark: | 70111b2a | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-2832/3/) | 2022-09-01 17:54:29 | ~4 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-220901-174941-70111b-pr2832.aar) |

alaibe reviewed (commented):
Curious to see how it renders in the app, last time I checked, there wasn't enough data to show preview.
Do u have a screenshot?

stefandunca commented:
> Curious to see how it renders in the app, last time I checked, there wasn't enough data to show preview. Do u have a screenshot?

I posted a screenshot here: https://github.com/status-im/status-desktop/pull/7202#issuecomment-1234129350

However, note that this is using the native QML image preview, a workaround you proposed discussed in the #desktop channel. Basically, it doesn't go through status-go preview system which would have made a bigger effort and probably the proper implementation using tenor preview API.

stefandunca commented:
> Curious to see how it renders in the app, last time I checked, there wasn't enough data to show preview. Do u have a screenshot?

@alaibe check my latest changes where I added a generic image preview and removed the hard-coding in QML

![image](https://user-images.githubusercontent.com/47554641/187983531-51d1a4cd-251c-47dd-a70d-fbecf1babbad.png)
